src/protocolsupport/protocol/typeremapper/itemstack/complex/ItemStackComplexRemapperRegistry.java

```diff
@@ -83,6 +83,7 @@ protected static void register(
 			registerToClient(material, enchantfilter, ProtocolVersionsHelper.ALL_PC);
 			registerToClient(material, enchanttolegacyid, ProtocolVersionsHelper.BEFORE_1_13);
 			registerToClient(material, dnametolegacytext, ProtocolVersionsHelper.BEFORE_1_13);
```

Comment:
register with BEFORE_1_13_AND_PE instead of second register (It doesn't make a difference in performance, just is preference)